src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+                            outputDataInConsole("Incorrect number, please enter correct number Room: ");
+                            continue;
+                        } else {
+                            String hotel = new ArrayList<>(foundRooms.keySet()).get(0);
+//                            hotelSet = Injector.getHotelController().findHotelByName(hotel).get(--hotelNumber);
+//                            roomSet = foundRooms.get(hotel).get(--roomNumber);
+//                            reserveId = Injector.getReservationController().bookRoom(reservedFrom,
+//                                    reservedTo, roomSet, hotelSet);
+                        }
+                    } catch (IOException e) {
+                        outputDataInConsole("Entered incorrect number hotel");
+                    }
+                    outputDataInConsole("Guest room reserved!\n You ID reservation: " + reserveId);
+                    outputSplitLine();
+                    break;
+                case (2):
+                    serviceMenu();
+                    bookFlaf = false;
+                    break;
+                default:
+                    continue;
+            }
+        }
+    }
+
+    private List<Reservation> getAllUserReservationsMenu() {
+        outputSplitLine();
+        List<Reservation> reservList = reservationController.getAllUserReservations();
+        reservList.forEach(item -> outputDataInConsole(item.toString() + "\n"));
+        return reservList;
+    }
+
+    private void cancelReservationMenu() {
+        Long idReservation = 0L;
+
+        try {
+            idReservation = (Long) inputDataFromUser("Enter your reservation number (to exit enter \"0\"): ", false,
+                    true, false, false);
+            boolean reservId = reservationController.cancelReservation(idReservation);
+            if (idReservation == 0) {
+                try {
+                    serviceMenu();
+                } catch (IOException e) {
+                    outputDataInConsole("Don't loaded Service menu" + e);
+                }
+            }
+            if (reservId) {
+                outputDataInConsole("Reserve a room removed!");
+                outputSplitLine();
+            } else {
+                outputDataInConsole("Entered incorrect number reservation!");
+                cancelReservationMenu();
+            }
+        } catch (IOException e) {
+            outputDataInConsole("Room reservations missing");
+        }
+    }
+
+    private void compareDate(Date dateFrom, Date dateTo) throws IOException {
+        boolean compareDate = true;
+        while (compareDate) {
+            if (dateFrom.compareTo(dateTo) <= 0) {
+                compareDate = false;
+            } else {
+                outputDataInConsole("Room reserved at least a day!");
+                dateTo = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "dateTo". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)